### ISSUE TITLE
Template button doesn't do anytyhing

### ISSUE THREAD

**Issue by Ipols0** [xaukael/ffs#12]
![Cattura](https://github.com/xaukael/ffs/assets/114725057/1a127949-112a-44af-a4ce-d79dcd4f0978)

Foundry is on 12.327
System is the official D&D 5e on 3.2.1
Freeform Sheets is on 1.17.5

I followed the steps on the english tutorial

**Comment by xaukael** [comment]
I just tested with those versions and it seems to be working correctly. Open the console (F12) and screenshot any errors you have there.

**Comment by Valdus92** [comment]
I have the same build and I can't get passed the same button. Here is the error messages I got. 

[localhost-1719580661021 error.txt](https://github.com/user-attachments/files/16030983/localhost-1719580661021.error.txt)

**Comment by Valdus92** [comment]
I have the same build and I can't get passed the same button. Here is the error messages I got. 

[localhost-1719580661021 error.txt](https://github.com/user-attachments/files/16030983/localhost-1719580661021.error.txt)

Also 

Device: 

Device name	sidecar
Processor	13th Gen Intel(R) Core(TM) i5-13420H   2.10 GHz
Installed RAM	16.0 GB (15.7 GB usable)
Device ID	FD0F2EE4-7395-4D3E-82D1-DE29E5825943
Product ID	00326-10000-00000-AA433
System type	64-bit operating system, x64-based processor
Pen and touch	No pen or touch input is available for this display

OS

Edition	Windows 11 Home
Version	23H2
Installed on	‎5/‎8/‎2024
OS build	22631.3737
Experience	Windows Feature Experience Pack 1000.22700.1009.0

Tried it on desktop, and by suggestion on firefox and edge. 

and this in the Dev tools 

![error message in f12](https://github.com/xaukael/ffs/assets/26496870/a2125006-7be3-4c70-a56d-27545778c136)
